### PR TITLE
Contact Tracing UI V1

### PR DESCRIPTION
**Pull request by adpirz**
Implements contact tracing and some of the updates to the UI, though not all (is not a full implementation of new header designs); I can revert so we keep existing styling and just add contact tracing, only weak spot is the `Beta` tags then

### TODO:
- Spot check states to see that formatting isn't broken
- Check all mobile states
- Remove unnecessary `TODO`s

image

image

## Scroll To (scroll is on click)

summaryscroll

### CODE REVIEW

**Comment by goldblatt** [comment]
ooo! this looks sooososo good a few over all comments. 

I think the home screen looks a lil squashed on mobile: 
Screen Shot 2020-05-15 at 2 49 07 PM

I think that the beta tag looks a lil off
Screen Shot 2020-05-15 at 2 49 25 PM

And finally 
I think if we have the auto scroll down would be good to have a scroll up to the top!

File: src/components/LocationPage/ChartsHolder.tsx [hunk 1]
```diff
@@ -146,6 +169,30 @@ const ChartsHolder = (props: {
                   </Disclaimer>
                 </>
               )}
+              <ChartHeader ref={contactTracingRef}>
+                {getMetricName(Metric.CONTACT_TRACING)}
+                <BetaTag>Beta</BetaTag>
+              </ChartHeader>
```
Comment (goldblatt) [inline src/components/LocationPage/ChartsHolder.tsx:175]: something about this seemed to have changed the location of the beta tag?

Comment (adpirz) [reply src/components/LocationPage/ChartsHolder.tsx:175]: Yeah, this is an intended change based on moving towards new design: 
image

https://www.figma.com/file/UUCr3ZpwaPV4o1KuGdSCNl/Covid?node-id=3985%3A3471

File: src/components/SummaryStats/SummaryStats.style.tsx [hunk 2]
```diff
@@ -18,13 +18,14 @@ export const SummaryStatsWrapper = styled(Box)<{ condensed?: Boolean }>`
...
     flex-direction: column;
+    cursor: pointer;
 
     @media (min-width: 600px) {
       flex-direction: row;
       border-radius: 2px;
       box-shadow: 0 1px 2px 1px rgba(0, 0, 0, 0.1);
       background-color: white;
-      padding: 1.75rem;
+      padding: 1.75rem 1;
```
Comment (goldblatt) [inline src/components/SummaryStats/SummaryStats.style.tsx:28]: shouls this be 1rem?

**Comment by goldblatt** [comment]
hmm the chart zones seem off still hmmm
Screen Shot 2020-05-18 at 4 10 27 PM

**Comment by mikelehen** [comment]
This got rolled into @goldblatt's PRs.

**Pull request closed without merging**